examples/text_processing/g2p/heteronym_classification_inference.py

```diff
@@ -150,8 +159,7 @@ def main(cfg):
             logging.info("No 'word_id' values found, skipping evaluation.")
             if os.path.exists(cfg.errors_file):
                 os.remove(cfg.errors_file)
-
-    if cfg.interactive:
+    else:
```

Comment:
Should probably still check if the user has set `cfg.interactive` and that we didn't just miss the manifest